docker/Dockerfile-pgbase

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
```

Comment:
I'd use `python:2.7-slim` (or even better, `python:2.7-slim-stretch`, which is based on the newer "Debian Stretch") - since they are 50MB compared to the standard image's 273MB (!).